Resync web-platform-tests/preload

rniwa opened the pull request:
#### 56a38429e20fdda4beadcf9865477d3a9a0a711e
<pre>
Resync web-platform-tests/preload
<a href="https://bugs.webkit.org/show_bug.cgi?id=252430">https://bugs.webkit.org/show_bug.cgi?id=252430</a>

Reviewed by Tim Nguyen.

Sync as of 5659625cf77d23579eeb41cd546b8763826db49d.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/preload/META.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-modulepreload-exec-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-modulepreload-exec.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-modulepreload-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-modulepreload.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-preload-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-preload-style-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-preload-style.html: Copied from LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-preload.html.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-preload.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-prefetching-on-text-plain-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-prefetching-on-text-plain-inner-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-prefetching-on-text-plain-inner.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-prefetching-on-text-plain-inner.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-prefetching-on-text-plain.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/download-resources-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/download-resources.html: Copied from LayoutTests/imported/w3c/web-platform-tests/preload/preload-default-csp.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/preload/dynamic-adding-preload-nonce-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/dynamic-adding-preload-nonce.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/dynamic-adding-preload-nonce.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-modulepreload-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-modulepreload.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-on-subresource-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-on-subresource.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-delay-onload-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-delay-onload.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-delay-onload.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-non-html-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-non-html.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-nonce-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-nonce.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload.html.headers:
* LayoutTests/imported/w3c/web-platform-tests/preload/modulepreload-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/modulepreload.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/onerror-event-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/onerror-event.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/onload-event.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/preconnect-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preconnect-onerror-event-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preconnect-onerror-event.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preconnect.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-accept-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-accept.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-cache-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-cache.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-events-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-events.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-headers-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-headers.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-load-event-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-load-event.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-types-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-types.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-connect-to-doc-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-connect-to-doc.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-csp.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-csp.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-default-csp.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-default-csp.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-dynamic-csp-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-dynamic-csp.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-error.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-error.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-font-crossorigin-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-font-crossorigin.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-in-data-doc-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-in-data-doc-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-in-data-doc.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-invalid-resources-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-invalid-resources.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-link-cached-stylesheet-different-doc-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-link-cached-stylesheet-different-doc.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-referrer-policy.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-resource-match.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-resource-match.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-strict-dynamic.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-strict-dynamic.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-time-to-fetch.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-time-to-fetch.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-type-match-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-type-match.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-with-type.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-xhr-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-xhr.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/reflected-as-value.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/cross-origin-module.py:
(main):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/echo-preload-header.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/echo-with-cors.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/font.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/font.ttf.sub.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/link-header-referrer-policy.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/module1.js:
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/module1.mjs: Copied from LayoutTests/imported/w3c/web-platform-tests/preload/resources/module1.js.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/modulepreload-iframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/prefetch-exec.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/prefetch-helper.js: Added.
(async get_prefetch_info):
(async prefetch):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/prefetch-info.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/preload_helper.js:
(stashPutUrl):
(encodedStashPutUrl):
(async hasArrivedAtServer):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/slow-exec.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/sound_5.oga.sub.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/square.png.sub.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/stash-put.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/stash-take.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/syntax-error.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/white.mp4.sub.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/single-download-preload.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/subresource-integrity-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/subresource-integrity-font-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/subresource-integrity-font.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/subresource-integrity-partial-image-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/subresource-integrity-partial-image.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/subresource-integrity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/w3c-import.log: Added.
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/260462@main">https://commits.webkit.org/260462@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/d211898615c6e544b908cd2a762fc984ac730427

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/108294 "7 style errors") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/17386 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/41159 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/117411 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/116795 "Built successfully") 
| | [  ~~🛠 ios-sim~~](https://ews-build.webkit.org/#/builders/76/builds/18916 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/8665 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/100511 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/114063 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/76/builds/18916 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/97346 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/42064 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/96084 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/28990 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/83758 "Found 1 new API test failure: /WebKitGTK/TestWebKitAccessibility:/webkit/WebKitAccessibility/text/state-changed (failure)") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/10225 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/30334 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/10964 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/7240 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/16374 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/49931 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/12550 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/3931 "Built successfully and passed tests") | | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/preload/link-header-modulepreload-expected.txt:
```diff
@@ -0,0 +1,6 @@
+
+Harness Error (TIMEOUT), message = null
+
+TIMEOUT test that a header-preloaded module is loaded and consumed Test timed out
```
It would be nice to skip tests that timeout to avoid slowing EWS, I know @cdumez did a bunch of work around that in november.

rniwa replied:
Done.

nt1m reviewed (approved):
Please include WPT revision hash in commit message.

rniwa committed:
Resync web-platform-tests/preload
https://bugs.webkit.org/show_bug.cgi?id=252430

Reviewed by Tim Nguyen.

Sync as of 5659625cf77d23579eeb41cd546b8763826db49d.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/preload/META.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-modulepreload-exec-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-modulepreload-exec.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-modulepreload-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-modulepreload.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-preload-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-preload-style-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-preload-style.html: Copied from LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-preload.html.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-delaying-onload-link-preload.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-prefetching-on-text-plain-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-prefetching-on-text-plain-inner-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-prefetching-on-text-plain-inner.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-prefetching-on-text-plain-inner.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/avoid-prefetching-on-text-plain.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/download-resources-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/download-resources.html: Copied from LayoutTests/imported/w3c/web-platform-tests/preload/preload-default-csp.sub.html.
* LayoutTests/imported/w3c/web-platform-tests/preload/dynamic-adding-preload-nonce-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/dynamic-adding-preload-nonce.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/dynamic-adding-preload-nonce.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-modulepreload-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-modulepreload.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-on-subresource-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-on-subresource.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-delay-onload-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-delay-onload.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-delay-onload.html.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-non-html-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-non-html.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-nonce-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload-nonce.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/link-header-preload.html.headers:
* LayoutTests/imported/w3c/web-platform-tests/preload/modulepreload-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/modulepreload.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/onerror-event-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/onerror-event.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/onload-event.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/preconnect-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preconnect-onerror-event-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preconnect-onerror-event.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preconnect.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-accept-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-accept.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-cache-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-cache.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-document-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-document.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-events-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-events.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-headers-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-headers.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-load-event-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-load-event.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-types-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/prefetch-types.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-connect-to-doc-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-connect-to-doc.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-csp.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-csp.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-default-csp.sub-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-default-csp.sub.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-dynamic-csp-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-dynamic-csp.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-error.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-error.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-font-crossorigin-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-font-crossorigin.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-in-data-doc-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-in-data-doc-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-in-data-doc.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-invalid-resources-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-invalid-resources.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-link-cached-stylesheet-different-doc-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-link-cached-stylesheet-different-doc.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-referrer-policy.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-resource-match.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-resource-match.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-strict-dynamic.sub-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-strict-dynamic.sub.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-time-to-fetch.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-time-to-fetch.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-type-match-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-type-match.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-with-type.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-xhr-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/preload-xhr.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/reflected-as-value.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/cross-origin-module.py:
(main):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/echo-preload-header.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/echo-with-cors.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/font.ttf: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/font.ttf.sub.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/link-header-referrer-policy.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/module1.js:
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/module1.mjs: Copied from LayoutTests/imported/w3c/web-platform-tests/preload/resources/module1.js.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/modulepreload-iframe.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/prefetch-exec.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/prefetch-helper.js: Added.
(async get_prefetch_info):
(async prefetch):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/prefetch-info.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/preload_helper.js:
(stashPutUrl):
(encodedStashPutUrl):
(async hasArrivedAtServer):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/slow-exec.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/sound_5.oga.sub.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/square.png.sub.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/stash-put.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/stash-take.py: Added.
(main):
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/syntax-error.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/resources/white.mp4.sub.headers: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/single-download-preload.html:
* LayoutTests/imported/w3c/web-platform-tests/preload/subresource-integrity-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/subresource-integrity-font-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/subresource-integrity-font.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/subresource-integrity-partial-image-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/subresource-integrity-partial-image.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/subresource-integrity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/preload/w3c-import.log: Added.
* LayoutTests/platform/gtk/TestExpectations:
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/260462@main
